DSCResources/VE_XD7Controller/VE_XD7Controller.psm1

```diff
@@ -147,7 +148,7 @@ function Set-TargetResource {
 
     } #end begin
     process {
-
+        ##Not sure if what I did above caused this, but when set to 'Absent' this block will remove the $existing controller instead of the one you're running the script from
```

Comment:
I don't think so and it looks like the wrong implementation and needs to be changed. Can you remove this comment and open an issue to get it fixed?